cprover_bindings/src/goto_program/typ.rs

```diff
@@ -308,6 +323,14 @@ impl Type {
         }
     }
 
+    pub fn size(&self) -> Option<u64> {
```

Comment:
Can we use `len()` instead? Can you also please add a comment to this method.

Reply:
Done.